src/components/Steps/Timeline.js

```diff
@@ -0,0 +1,114 @@
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+import styles from "../../styles/Home.module.css";
+
+import Select from "react-select";
+import {
+  HasSyncedOptions,
+  InfiniteSessionsOptions,
+  IsEnabledOptions,
+} from "../Options/TimelineOptions";
+
+export default function Timeline({ setActiveStep }) {
```

Comment:
Add comment explaining what this method represents in the UI. 

Actually, do the same for all the components

Reply:
Okay adding them